From ec6a728e9ddc8d773d4b206bb07bd0155509fb88 Mon Sep 17 00:00:00 2001
From: toidiu <apoorv@toidiu.com>
Date: Wed, 30 Oct 2024 14:40:05 -0700
Subject: [PATCH] patch toggle

---
 tls/s2n_security_policies.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tls/s2n_security_policies.c b/tls/s2n_security_policies.c
index 3a331b9c6..1bfef6d9e 100644
--- a/tls/s2n_security_policies.c
+++ b/tls/s2n_security_policies.c
@@ -1218,9 +1218,9 @@ const struct s2n_security_policy security_policy_null = {
 };
 
 struct s2n_security_policy_selection security_policy_selection[] = {
-    { .version = "default", .security_policy = &security_policy_20240501, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "default", .security_policy = &aecurity_policy_20240501, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "default_tls13", .security_policy = &security_policy_20240503, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
-    { .version = "default_fips", .security_policy = &security_policy_20240502, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "default_fips", .security_policy = &aecurity_policy_20240502, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "default_pq", .security_policy = &security_policy_20241001, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "20240501", .security_policy = &security_policy_20240501, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "20240502", .security_policy = &security_policy_20240502, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
-- 
2.39.5 (Apple Git-154)

